### DISCUSSION TITLE
Anonymized DNS

### DISCUSSION THREAD

**Discussion by drtiwarir** [DNSCrypt/dnscrypt-proxy#2085] [Q&A]
I configured Dnsmasq and Dnscrypt on one DigitalOcean droplet and a relay server on another droplet.
It works well, but does not hide my IP address. I am planning to use it for geoblocked streaming videos.
As per documentation "Instead of directly reaching a server, an Anonymized DNS client encrypts the query for the final server, but sends it to a relay.
The relay doesn't know the secret key, and cannot learn anything about the content of the query. It can only blindly forward the query to the actual DNS server, the only server that can decrypt it.
The DNS server itself receives a connection from the relay, not from the actual client. So the only IP address is knows about is the one of the relay, making it impossible to map queries to clients"

Can somebody provide me more information about how to fix this ?
Thanks for your help.

**Comment by jedisct1** [comment]
It's very likely that the video streaming website is using more than DNS in order to check for your location.

**Reply by drtiwarir** [reply]
Even YouTube is detecting my original IP address and shows me the content of my current location.
Ideally all websites I visit must see my relay server’s IP address 
Maybe I am missing something. Can you please provide me brief guide/tutorial 

Thanks

**Reply by lifenjoiner** [reply]
The relay only hide you from the upstream DNS servers, rather than the website you visit. It is not a HTTP or SOCKS proxy. They are totally 2 different routes.
